src/main/java/uk/gov/hmcts/reform/sscs/transformers/SscsCaseTransformer.java

```diff
@@ -122,18 +122,20 @@ public CaseResponse transformExceptionRecord(ExceptionRecord exceptionRecord, bo
             return formTypeValidatorResponse;
         }
 
-        String formType = exceptionRecord.getFormType();
-        log.info("formtype for case {} is {}", caseId, formType);
 
-        if (formType == null || notAValidFormType(formType)) {
-            ScannedData scannedData = sscsJsonExtractor.extractJson(exceptionRecord);
-            String ocrFormType = getField(scannedData.getOcrCaseData(), FORM_TYPE);
+        ScannedData scannedData = sscsJsonExtractor.extractJson(exceptionRecord);
+        String formType = getField(scannedData.getOcrCaseData(), FORM_TYPE);
+
+        if (formType == null  || notAValidFormType(formType)) {
+            formType = exceptionRecord.getFormType();
 
-            if (!notAValidFormType(ocrFormType)) {
-                formType = ocrFormType;
+            if (formType == null || notAValidFormType(formType)) {
```

Comment:
Can you change to formType != null && notAValidFormType(formType) as there is no need to set the field to null if already nul

Reply:
same as above